### ISSUE TITLE
Risks of doing UAT/Dev in production

### ISSUE THREAD

**Issue by pkbullock** [MicrosoftDocs/microsoft-365-community#514]
#### Article suggestion

Based on the Twitter thread, around uat/dev in prod. 

Whilst it can be done, there are reasons and risks associated with doing so. 

Listing these out. Explaining why separate tenants would be better practice.

**Comment by eemancini** [comment]
Are you interested in authoring this article, @pkbullock?

**Comment by pkbullock** [comment]
Yeap, sorry these dropped off the radar. Thx for highlighting.

**Comment by eemancini** [comment]
No worries at all. We know this is volunteered time and we appreciate your contributions :)

**Comment by tseward** [comment]
@pkbullock doesn't it depend on what dev/UAT you're performing in production? I'd gather most businesses do not have multiple tenants for this purpose. I think this was important in SharePoint Server due to full trust deployments possibly fundamentally breaking the farm, but given we can only mostly make "content changes" with SPFx, those packages can be tested in "dev/UAT" site collections before being deployed on the 'production' site collection.

Is this another one of those "it depends" with SharePoint? :-)

**Comment by pkbullock** [comment]
Yes, more so now, sorry this dropped off my radar. I will plan to write up this month or sooner. If there are other interested contributors, I am happy to start writing a piece and others weigh in. I’m sure there are a few perspectives on this topic - it’s worth including those as part of the help to the reader, so they can make the informed decision.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tseward** [comment]
@pkbullock would be happy to weigh in.
